### PR TITLE
Slides for Langdev 2024

### PR DESCRIPTION
**Pull request by enikao**

### CODE REVIEW

**Comment by enikao** [comment]
We should squash merge this PR, to minimize redundant binary data in the repo.

File: resources/LangDev 2024 Talk-jos.pptx
Comment (enikao) [inline resources/LangDev 2024 Talk-jos.pptx:1]: @joswarmer I think we can delete this one?

File: resources/SEN-symposium-poster-layers.afpub
Comment (dslmeinte) [inline resources/SEN-symposium-poster-layers.afpub:1]: What format is this?

File: resources/LangDev 2024 Talk.pptx
Comment (dslmeinte) [inline resources/LangDev 2024 Talk.pptx:1]: Maybe we can go over all files in this directory and give them really descriptive file names? We've got a `LangDev talk.pptx` _and_ a `LangDev 2024 Talk.pptx` file now.

**Review by dslmeinte — CHANGES_REQUESTED** [review]

**Pull request merged**